src/model/roof.fixtures.ts

```diff
@@ -0,0 +1,18 @@
+import type { RoofSurfaceModel } from './roof.model'
+
+export const roofSurfaceFixtures = (): RoofSurfaceModel[] => [
+  {
+    values: [10, 20, 30, 40],
+    favorable: 44,
+    total: 90,
+    orientation: 'Sud',
+    incliniasion: 47.9,
+  },
+  {
+    values: [10, 20, 30, 40],
+    favorable: 44,
+    total: 90,
+    orientation: 'Sud',
+    incliniasion: 47.9,
+  },
```

Comment:
Maybe set different values for the second accordion , to see the difference with the first

Reply:
hehe, thanks